src/cc/lib/graph/metadata.cc

```diff
@@ -17,7 +17,7 @@ namespace snark
 {
 
 Metadata::Metadata(std::filesystem::path path, std::string config_path, bool skip_feature_loading,
-                   std::shared_ptr<Logger> logger)
+                   bool skip_watermark_loading, std::shared_ptr<Logger> logger)
```

Comment:
It seems like the goal is to take some data that is generated as temporal and turn it to a regular non-temporal dataset? Can we adjust the parameter name accordinglty so its more clear, probably no one will know what a watermark is

Reply:
> It seems like the goal is to take some data that is generated as temporal and turn it to a regular non-temporal dataset?
Yes, our data pipelines currently generate everything with temporal edges/features. Our temporal edges are consuming around 40% of our GE memory, which most of our models do not utilize. Therefore, we want to have the option to skip loading the temporal dataset for models that do not require it.

I renamed the parameter to skip_temporal_loading.